source/slang/slang-ir-serialize-types.h

```diff
@@ -22,36 +21,29 @@ enum class IRSerialCompressionType : uint8_t
 
 class StringRepresentationCache
 {
+    // TODO: The name of this type is no longer accurate to its function.
```

Comment:
Do we plan to rename it in a future PR?

Reply:
That would be my goal, yeah. I like to avoid putting bulk renaming/refactoring changes in the same PR as functionality changes, since it makes it harder to review.

A small PR that requires line-by-line review is usually easy.

A large PR that only has renaming changes is also easy to review.

The combination of the two is usually not easy to review.